pkgs/tools/misc/ili2c/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, jdk, ant, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  name = "ili2c-${version}";
```

Comment:
Please use `pname = "ili2c";` instead of `name`.

Reply:
Will I then have to build the `rev` string (which interestingly follows the ` "${pname}-${version}"` pattern) explicitly instead?

Reply:
Yes, unfortunately. Using `pname` is quickly becoming the convention, though certainly not mandatory.

Reply:
I've changed it anyway and now explicitly use `rev = "${pname}-${version}";`. See https://github.com/NixOS/nixpkgs/compare/b6388f7be50dbb30e56b18acb727fcabd0379fb5..2aaa8876ff68e454ff60fc9a2b6d2174df6bb17a#diff-efa52252ac746605798baf0ce1e25f13R12

I'm now using just the `pname` instead of the full name to determine where in `$out/share/` the JAR should go: https://github.com/NixOS/nixpkgs/compare/b6388f7be50dbb30e56b18acb727fcabd0379fb5..2aaa8876ff68e454ff60fc9a2b6d2174df6bb17a#diff-efa52252ac746605798baf0ce1e25f13R21